packages/react-dom/src/__tests__/ReactDOMAttribute-test.js

```diff
@@ -54,8 +54,8 @@ describe('ReactDOM unknown attribute', () => {
       expectDev(
         normalizeCodeLocInfo(console.error.calls.argsFor(0)[0]),
       ).toMatch(
-        'Warning: Received `true` for non-boolean attribute `unknown`. ' +
-          'If this is expected, cast the value to a string.\n' +
+        'Warning: Received `true` for non-boolean attribute `unknown`. If this is expected, cast ' +
```

Comment:
This change is not unnecessary, right? It just reformats the message in a slightly less readable way.